artwork/linedrawings/aaron.jsx

```diff
@@ -0,0 +1,182 @@
+import React from "react";

```

Comment:
No need to import React when using JSX through Webpack, which is how we'll load this.

I realize that's hard to know but I'm adding these comments to explain :)